draw/getsubfont_plan9.go

```diff
@@ -0,0 +1,27 @@
+// +build plan9
```

Comment:
you can elide the build directive here - having _plan9.go file suffix is sufficient.